backend/db/data/books/1.json

```diff
@@ -10,71 +10,71 @@
     },
     {
       "title": "The Journey from Platform Nine and Three-Quarters",
-      "summary": ""
+      "summary": "Harry is at King's Cross Station with the Dursleys, preparing to catch the train to Hogwarts. However, they are unable to find Platform Nine and Three-Quarters and the Dursleys abandon Harry on the station.  Fortunately, Harry overhears a woman talking about Muggles and asks her for help in finding the hidden platform. Mrs. Weasley tells him to run into the barrier between platforms and he miraculously pass through it finally finding the train to Hogwarts. This unexpected encounter with Mrs. Weasley and her family marks the beginning of Harry's magical journey. He spends the trip to Hogwarts with the youngest Weasley, Ron. Additionally, Harry meets Draco Malfoy, a fellow student on the Hogwarts Express, who leaves a distinct impression with his aloof and arrogant demeanor."
     },
     {
       "title": "The Sorting Hat",
-      "summary": ""
+      "summary": "Harry arrives at Hogwarts School of Witchcraft and Wizardry and is sorted into his house during the Sorting Hat ceremony. The hat briefly considers placing him in Slytherin, but ultimately chooses Gryffindor after Harry expresses a strong desire not to be in Slytherin. Throughout the ceremony, Harry becomes friends with Ron Weasley, who is also sorted into Gryffindor, and is introduced to the concept of the four houses at Hogwarts. Harry also has his first encounters with several of the Hogwarts teachers, including Professor Albus Dumbledore, Professor Severus Snape, Professor Minerva McGonagall, and the quirky Professor Quirrell, who has a turban concealing a secret."
     },
     {
       "title": "The Potions Master",
-      "summary": ""
+      "summary": "Harry attends his first Potions class at Hogwarts, taught by the enigmatic Professor Severus Snape. He quickly discovers that Snape has a strong dislike for him, leading to a tense and challenging class experience. Harry also experiences other classes, such as Transfiguration taught by Professor McGonagall and Charms with Professor Flitwick. Furthermore, Harry becomes aware of the break-in at Gringotts bank. "
     },
     {
       "title": "The Midnight Duel",
-      "summary": ""
+      "summary": "Harry attends his first Flying lesson, where he gets his first taste of Quidditch, the wizarding sport he quickly becomes passionate about. Harry's remarkable talent on a broomstick is noticed by Professor McGonagall, who recommends him for the Gryffindor Quidditch team. The tension between him and Draco Malfoy steadily escalates. Harry, Ron, and Hermione sneak out of their dormitories at night to challenge Malfoy to a duel. Harry and his friends do not encounter Malfoy, but instead, they stumble upon a massive three-headed dog guarding a mysterious trapdoor."
```

Comment:
Maybe we could have it more like: 
During his first flying lessons, hold by Madam Hooch, Harry's remarkable talent...

Also the connection between this event and the midnight duel is somehow missing.